templates/release/de/meter/cg-em24_0.yaml

```diff
@@ -1,6 +1,8 @@
 product:
   brand: Carlo Gavazzi
   description: EM24
+description: |
+ EM24 mit RS-485 anschluss. Benutze die EM24_E1 für die EM24 mit Ethernet anschluss, denn die Definition is nicht compatibel.
```

Comment:
EM24 mit RS-485 Anschluss. Benutze die EM24_E1 für die EM24 mit Ethernet-Anschluss, denn die Definition is nicht kompatibel.

Anschluss with capital letter.
Then with "-" or without I think both is allowed.
compatible -> kompatibel